src/utils/LoggingUtil.ts

```diff
@@ -0,0 +1,32 @@
+import { TEST_STATUS, TEST_RESULT } from "../assets/Enums";
+import { Defect, TestType } from "../models/ITestResult";
+import { ITestResultPayload } from "../models/ITestResultPayload";
+
+export class LoggingUtil {
```

Comment:
Since we agree that this is a temporary solution, maybe adding a comment with the future ticket number removing this code since we don't use feature switch would be a good idea so the next developers will know it is temporary and will be removed

Reply:
There is a ticket for the removal (https://jira.dvsacloud.uk/browse/CVSB-19020). I'd rather keep the removal tasks in jira than in comments but if you think that would be useful, I can add it

Reply:
Done